WearSpeakerSample/README.md

```diff
@@ -3,17 +3,16 @@ Android WearSpeakerSample Sample
 ================================
 
 A sample that shows how you can record voice using the microphone on a wearable and
-play the recorded voice or an mp3 file, if the wearable device has a built-in speaker.
+play the recorded voice or an mp3 file, if the wearable device is connected to a speaker
+(bluetooth or built-in).
 
 This sample doesn't have any companion phone app so you need to install this directly
-on your watch (using "adb").
+on your watch (using "adb" or Android Studio).
```

Comment:
Optional/unrelated, but maybe we could put adb in backticks instead of quotes, and the parentheses are probably unnecessary.

Reply:
Updated in latest push!